src/main/java/io/confluent/idesidecar/restapi/util/WebClientFactory.java

```diff
@@ -39,6 +39,7 @@ public class WebClientFactory {
   static final String CERT_HEADER = "-----BEGIN CERTIFICATE-----" + LINE_SEPARATOR;
   static final String CERT_FOOTER = LINE_SEPARATOR + "-----END CERTIFICATE-----" + LINE_SEPARATOR;
   static final List<String> WINDOWS_TRUST_STORE_NAMES = List.of("WINDOWS-MY", "WINDOWS-ROOT");
+  static final Integer CONNECT_TIMEOUT = 10000;
```

Comment:
Actually, should we add this as a config in `application.yaml`?

Reply:
Yeah, I've moved it.